spec/requests/can_respond_with_collection_of_latest_experiences_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe 'GET /api/articles', type: :request do
+  let!(:test_story) { create(:article, title: "Story Article") }
+  let!(:test_article1) { create(:article, article_type: 0, created_at: Time.current - 5.days) }
+  let!(:test_article2) { create(:article, title: 'Test Article 2', article_type: 0, created_at: Time.current - 2.days) }
+  let!(:test_article3) { create(:article, article_type: 0, created_at: Time.current - 3.days) }
+  let!(:test_article4) { create(:article, article_type: 0, created_at: Time.current - 4.days) }
+  let!(:test_article5) { create(:article, title: 'Test Article 5', article_type: 0, created_at: Time.current - 1.days) }
+  describe 'successfully' do
+    before do
+      get '/api/articles',
+          params: {
+            article_type: 'experience'
+          }
+    end
+
+    it 'responds with a 200 status' do
+      expect(response).to have_http_status 200
+    end
+
+    it 'filters out experience article, thus responding with a list of 5 articles' do
+      expect(response_json['articles'].length).to eq 5
+    end
+
+    it 'responds with newest article first in the list' do
+      expect(response_json['articles'].first['title']).to eq 'Test Article 5'
+    end
+
+    it 'responds with second newest article second in the list' do
+      expect(response_json['articles'].second['title']).to eq 'Test Article 2'
+    end
+
+    it 'the responded articles are expected not to contain the body attribute' do
+      expect(response_json['articles'].first['body']).to eq nil
```

Comment:
this test makes no sense, if you don't send that then you don't have to test that it is nil. When we test we want to check that the right information is being sent, not what is not being sent 👍🏻

Reply:
Not even if part of the functionality is filtration? Like the purpose is to NOT have it there :P

Reply:
No, in that case it would mak more sense to set an expected response and test that the response is equal to that rather than testing that something doesn't exist